components/areas/AlertWidget.js

```diff
@@ -20,7 +20,7 @@ import AreaSubscriptionModal from 'components/modal/AreaSubscriptionModal';
 import LayerInfoModal from 'components/modal/layer-info-modal';
 
 // Services
-import AreasService from 'services/AreasService';
+import { getGeostore } from 'services/areas';
```

Comment:
Geostore operations should have its own service.

Reply:
done